src/standalone/Vegetation/pmodel.jl

```diff
@@ -79,21 +85,28 @@ Base.@kwdef struct PModelConstants{FT}
     bS_Jmax::FT
     """Molar mass of carbon (kg mol^-1)"""
     Mc::FT
-    """Intercellular O2 mixing ratio (mol mol^-1)"""
+    """Intercellular O2 mixing ratio (unitless)"""
     oi::FT
     """First order coefficient for temp-dependent Rd (K^-1)"""
     aRd::FT
     """Second order coefficient for temp-dependent Rd (K^-2)"""
     bRd::FT
     """Constant factor appearing the dark respiration term for C3 plants (unitless)"""
     fC3::FT
+    """Planck constant (J s)"""
```

Comment:
This is minor, but we should have access to the `earth_param_set` wherever these are needed. can we get them from there, instead of duplicating?